src/components/TenantSelector/TenantSelector.tsx

```diff
@@ -192,18 +192,29 @@ class TenantSelector extends React.Component<
     this.setState({
       validity: TenantValidity.CHECKING,
     });
-    validateTenant(tenant, advancedOptions)
-      .then(() => {
+
+    try {
+      const isTenantValid = validateTenant
+        ? await validateTenant(tenant, advancedOptions)
```

Comment:
This will throw an exception if `validateTenant` is not defined?
And this will run https://github.com/cognitedata/gearbox.js/pull/89/files#diff-14f16035ee023b72b8993bcb65372f41R212 which will tell that the tenant is not valid?

Reply:
Fixed in 4f54f94b7642908db3a570184cee865bd53158a8 commit